scripts/lib/LinkChecker.ts

```diff
@@ -117,6 +117,52 @@ export class Link {
     );
   }
 
+  /**
+   * Returns a string with a suggested replacement for a broken link
+   */
+  didYouMean(existingFiles: File[], originFile: string): string {
+    const levenshtein = require("fast-levenshtein");
+
+    // Find a new valid link
+    let min_score = Number.MAX_SAFE_INTEGER;
```

Comment:
In JavaScript, the convention is to use `camelCase`. Ditto the other variables

Reply:
Done!